### DISCUSSION TITLE
Jupyter Server

### DISCUSSION THREAD

**Discussion by sofie-makovets** [cooperrc/engineering-dynamics#5] [Q&A]
I'm having a little trouble accessing the Jupyter Server. Anytime I try clicking on the link at the top of the syllabus it says that the certificate for the website is not valid and that there is a security error. A screenshot of what I see is shown below. Was wondering if anyone was going through something similar and what they have done to fix it. Thanks!
<img width="748" alt="Screenshot 2024-01-17 192045" src="https://github.com/cooperrc/engineering-dynamics/assets/156958761/b70b97a8-7dd1-43b3-9ab4-cdb629a3c523">

**Comment by aidanfurlonge** [comment]
You probably figured it out already but I just clicked Advanced then Allow Access
